src/main/java/com/hometask/task_security/controller/EmailController.java

```diff
@@ -0,0 +1,71 @@
+package com.hometask.task_security.controller;
+
+import com.hometask.task_security.model.Email;
+import com.hometask.task_security.repository.db_repo.JDBCRepoImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+
+@Controller
+@RequestMapping(value = "/", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
You do no need to produce JSON with this controller. It returns only Thymeleaf pages.